src/icons/AllAppsMenuIcon.js

```diff
@@ -1,57 +1,64 @@
-import React from 'react';
+import { BaseSvg } from 'icons';
 
-const svg = ({ width = '12px', height = '12px', ...props }) => (
-  <svg
+export default ({ fill = '#a6278f', width = 12, height = 12, marginRight = 10, ...props }) =>
+  BaseSvg({
+    svg: `<svg
     id="all_topics_purple"
     data-name="all topics purple"
     xmlns="http://www.w3.org/2000/svg"
     viewBox="0 0 13 13.1"
-    width={width}
-    height={height}
-    css={`
-      margin-right: 10px;
-    `}
-    {...props}
   >
-    <defs />
+    <defs >
+      <style>.cls-1{fill: ${fill}</style>      </defs>
```

Comment:
I'm a bit paranoid of using class name here, simply because of potential leaks. I think all the svg we get come with that `cls-1` class so if we allow that classname, they can mess with each other. Please correct me if I'm plain wrong, but a solution could be to put the fill as props for each of the `path`? :)

Reply:
For reference :)
 
Once we keep wrapping our SVGs in our BaseSVG class we are good because it turns it into an image with a data URL source